SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -28,36 +29,38 @@ public void AddButton()
         }
 
         /// <summary>
-        ///  txtbox Emp ID and insert stored ID
+        /// this is the second time on page where we...... 
+        /// search for emp id textbox and insert stored ID
+        /// then we click search button
         /// </summary>
         public void SetTxtBoxEmpID(string storeEmployeeID)
         {
             EditBoxSendKeysAndVerify(_employeeIDTxtBoxLocator, storeEmployeeID);
+            Click(_searchButtonLocator);
         }
 
-        public void SearchButton()
+        /// <summary>
+        ///  this is to verify ID # in search result
+        /// </summary>
+        public string verifyEmployeeId()
         {
-            Click(_searchButtonLocator);
+            return GetText(_actualIDLocator);
         }
 
         /// <summary>
-        ///  to verify ID # in search result
+        ///  this is to verify first name in search result
         /// </summary>
-        public void verifyEmployeeId(string employeeId)
+        public string verifyFirstNameResults()
```

Comment:
This method name no longer matches what it's doing.

Reply:
I did rename it to VerifyFirstNameResults since we are grabbing the firstname in the search results so we can verify it does match